### PR TITLE
42 add billing portal endpoint

### PR DESCRIPTION
**Pull request by martmull**
- add billingPortalSession endpoint to billing resolver
- rename checkout to checkoutSession

### CODE REVIEW

File: packages/twenty-server/src/core/billing/billing.resolver.ts [hunk 1]
```diff
@@ -38,11 +39,25 @@ export class BillingResolver {
...
+  @Query(() => SessionEntity)
   @UseGuards(JwtAuthGuard)
-  async checkout(
+  async billingPortalSession(
     @AuthUser() user: User,
-    @Args() { recurringInterval, successUrlPath }: CheckoutInput,
+    @Args() { returnUrlPath }: BillingSessionInput,
+  ) {
+    return {
+      url: await this.billingService.billingPortalSession(
```
Comment (charlesBochet) [inline packages/twenty-server/src/core/billing/billing.resolver.ts:49]: can we rename this method on billingService to computeBillingPortalSessionURL (same for computeCheckoutSessionURL)?

File: packages/twenty-server/src/core/billing/dto/billing-session.input.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { ArgsType, Field } from '@nestjs/graphql';
+
+import { IsOptional, IsString } from 'class-validator';
+
+@ArgsType()
+export class BillingSessionInput {
+  @Field(() => String, { nullable: true })
+  @IsString()
+  @IsOptional()
```
Comment (charlesBochet) [inline packages/twenty-server/src/core/billing/dto/billing-session.input.ts:9]: is it optional?

Comment (martmull) [reply packages/twenty-server/src/core/billing/dto/billing-session.input.ts:9]: yes it is

**Review by charlesBochet — APPROVED** [review]
LGTM!

**Pull request merged**